### PR TITLE
[ticket/14145] Add quickreply_editor_subject_after event

### PR DESCRIPTION
**Pull request by Senky**
[PHPBB3-14145](https://tracker.phpbb.com/browse/PHPBB3-14145)

### CODE REVIEW

**Comment by Zoddo** [comment]
You need to remove the merge commit to fix tests.
```
git checkout ticket/14145
git fetch upstream
git reset --hard upstream/3.1.x
git cherry-pick ce43a61
git push -f origin ticket/14145
```

Also applies for three others PRs.

File: phpBB/styles/prosilver/template/quickreply_editor.html [hunk 1]
```diff
@@ -8,6 +8,7 @@ <h2 class="quickreply-title">{L_QUICKREPLY}</h2>
 						<dt><label for="subject">{L_SUBJECT}{L_COLON}</label></dt>
 						<dd><input type="text" name="subject" id="subject" size="45" maxlength="124" tabindex="2" value="{SUBJECT}" class="inputbox autowidth" /></dd>
 					</dl>
+				<!-- EVENT quickreply_editor_subject_after -->
 				<!-- EVENT quickreply_editor_message_before -->
```
Comment (Zoddo) [inline phpBB/styles/prosilver/template/quickreply_editor.html]: Why don't use `quickreply_editor_message_before`?

Comment (Senky) [inline phpBB/styles/prosilver/template/quickreply_editor.html]: I am not against it, I just included this event for convention, that every _before event has it's _after event, too.

File: phpBB/docs/events.md [hunk 2]
```diff
@@ -1283,6 +1283,13 @@ quickreply_editor_message_before
 * Since: 3.1.0-a4
 * Purpose: Add content before the quick reply textbox
 
+quickreply_editor_subject_after
+===
+* Locations:
+    + styles/prosilver/template/quickreply_editor.html
+* Since: 3.1.7
```
Comment (Zoddo) [inline phpBB/docs/events.md]: Should be `3.1.7-RC1` here, no @Nicofuma?

Comment (Nicofuma) [inline phpBB/docs/events.md]: yes, it should

File: phpBB/styles/prosilver/template/quickreply_editor.html (hunk 1 above)
Comment (marc1706) [inline phpBB/styles/prosilver/template/quickreply_editor.html]: In this case it doesn't have any purpose though.

**Pull request closed without merging**